common/djangoapps/xblock_django/user_service.py

```diff
@@ -65,6 +65,12 @@ def get_current_user(self):
         """
         return self._convert_django_user_to_xblock_user(self._django_user)
 
+    def get_current_django_user(self):
+        """
+        Returns the currently-logged in user, as an instance of Django AUTH_USER_MODEL
+        """
+        return self._django_user
```

Comment:
It seems like up to now, the User Service was explicitly designed _not_ to expose the Django user to XBlocks. This would allow XBlocks to do all kinds of things, for example change `user.is_superuser = True; user.save()` which we probably don't want to allow as part of the public API.

What do you think about just passing the user to the `EnrollmentsService` when it's initialized in the XBlock runtime, and then making the `user` parameter optional? If omitted, the EnrollmentsService can just use the current user which was specified when the services was initialized.

Reply:
Yes that makes sense.